src/Microsoft.DotNet.Interactive.ExtensionLab/SQLiteKernelConnector.cs

```diff
@@ -22,5 +22,9 @@ public SQLiteKernelConnector(string connectionString)
         {
             ConnectionString = connectionString;
         }
+
+        public void Dispose()
```

Comment:
Why the empty `Dispose` methods on several of these?

Reply:
the connector for sql and kql doesn't need disposing, the toolprocess is controlled by the kernel